scripts/generate_presign_url/README.md

```diff
@@ -0,0 +1,18 @@
+# Generate Presign URL
+
+This script is to be used to create pre-signed url based on the below parameter. Pre-signed url will live for 7 days.
+
```

Comment:
I guess the default use will be from the command line, by a user that is not familiar with the agha setup.

Perhaps add a section on how to download/install... (e.g. git clone or partial checkout, python requirements, ...)

Reply:
Yes agree, added a requirements.txt, and update on readme including the setup instructions.